Resources/Prototypes/Entities/Objects/Weapons/Guns/Shotguns/shotguns.yml

```
@@ -138,7 +138,8 @@
   name: Enforcer
   parent: [BaseWeaponShotgun, BaseGunWieldable, BaseRestrictedContraband]
   id: WeaponShotgunEnforcer
-  description: A premium combat shotgun based on the Kammerer design, featuring an upgraded clip capacity. .50 shotgun shells.
+  description: The Enforcer was designed as a modern remake of the Kammerer design. It features new polymer furniture and an extended magazine tube. After being presented to NT it was quickly adopted as the new shotgun for station security teams, but as production lines have not yet been re-tooled to produce them in bulk, most stations have only received them in limited numbers.
```

```suggestion
  description: The Enforcer was designed as a modern remake of the Kammerer design. It features new polymer furniture and an extended magazine tube.
```
too much RP in the description

---

Yeah I wanted some excuse as to why its not standard-issue, ill remove the bloat

---

hows this?
"The Enforcer was designed as a modern remake of the Kammerer design, featuring new polymer furniture and an extended magazine tube. It has yet to see mass production and as such is somewhat rare among security teams."